streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -713,7 +713,7 @@ private void addStreamTasks(Collection<TopicPartition> assignment) {
                     for (TopicPartition partition : partitions)
                         activeTasksByPartition.put(partition, task);
                 } catch (StreamsException e) {
-                    log.error("{} Failed to create an active task %s: ", logPrefix, taskId, e);
+                    log.error(String.format("%s Failed to create an active task %s: ", logPrefix, taskId), e);
```

Comment:
It's a bit confusing, but it can: http://slf4j.org/faq.html#paramException (if I understood your point correctly).